mesh.proto

```diff
@@ -1166,6 +1166,19 @@ message LogRecord {
   Level level = 4;
 }
 
+message QueueStatus {
+  /* Last attempt to queue status */
+  uint32 res = 1;
```

Comment:
If i read the code correctly, res is of type ErrorCode which is int. Please use an option to constrain to int8 here too...